content/pages/sprint/2021/06early-devenv.rst

```diff
@@ -0,0 +1,72 @@
+=====================================================================
+Scisprint 2021 Early of June: solvcon devenv
+=====================================================================
+
+:date: 2021-05-24 20:46
+:url: sprint/2021/06early-devenv
+:save_as: sprint/2021/06early-devenv.html
+
+Scisprint 2021 Early of June: solvcon devenv
+=====================================================================
+
+We cordially invite you to join the short sprint for the development
+environment for `solvcon <https://solvcon.net/>`_ projects.  solvcon_ is a host
+of projects focusing on numerically solving the governing equations of
+transient phenomena.
+
+The development environment, i.e., devenv (see `the github page
+<https://github.com/solvcon/devenv>`__), is also a solvcon project, but has a
+much broader scope that includes the infrastructure for developing code for
+numerical calculations, which are the foundation of computational sciences and
+mathematics.  The real-world applications, including physics, mechanics,
+semiconductor, finance, machine learning, etc., will be built with it.
+
+Numerical calculations require writing high-performance code.  The complex
+implementations call for tremendous amount of code to be written and it is not
+possible to do it all by a single team.  External dependencies are necessary.
+However, because of the requirements of accuracy and performance, the
+dependencies are usually at the source level.  Building dependencies from
+source is the norm.  The project devenv is made to help the source-level
+dependency management and alignment for the runtime among developers.
+
+Registration
+------------
+
+Please RSVP at:
```

Comment:
This time we don't have rsvp url. How about put sciwork gather.town url on here?

Reply:
I second the idea of using the gather.town link.

Reply:
Agree !!